<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoShower</class>
 <widget class="QWidget" name="UserInfoShower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserInfoShower</string>
  </property>
  <widget class="QLabel" name="m_img">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pwd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
